docs/guides/preinstall.md

```diff
@@ -13,15 +13,15 @@ In macOS Disk Utility, [create a partition](https://support.apple.com/guide/disk
 
 ### Selecting an ISO
 
-If there is an ISO with T2 support for your distro, you can download it here:
+Listed below are the currently available ISO files with T2 support. Choose a distribution and follow the download link:
```

Comment:
```suggestion
If there is an ISO with T2 support for your distro, you can download it here:
```

This change makes it much longer without improving clarity. The original sentence is already simple enough.

Reply:
```suggestion
Listed below are the currently available ISO files to download:
```
I think it's more well written than "If there is.."

Reply:
`Listed below are the currently available installer ISOs for download:` would be better